mtest/operators.go

```diff
@@ -332,66 +331,6 @@ func TestOperators(isDegraded bool) {
 		cluster.Options.Kubelet.ExtraEnvvar = map[string]string{"AAA": "aaa"}
 		cluster.Options.Kubelet.Domain = "neconeco"
 		clusterSetAndWait(cluster)
-
-		By("Adding a scheduler extender")
-		// this will run these ops:
-		// - SchedulerRestartOp
-		cluster.Options.Scheduler.Extenders = []string{"urlPrefix: http://127.0.0.1:8000"}
-		clusterSetAndWait(cluster)
-
-		stdout, stderr, err := execAt(node1, "jq", "-r", "'.extenders[0].urlPrefix'", op.PolicyConfigPath)
-		Expect(err).NotTo(HaveOccurred(), "stdout=%s, stderr=%s", stdout, stderr)
-		Expect(strings.TrimSpace(string(stdout))).To(Equal("http://127.0.0.1:8000"))
```

Comment:
Didn't confirmed values in PolicyConfigPath at strategy test.go
Don't you need to check the value of the changed result for each case?
https://github.com/cybozu-go/cke/blob/master/server/strategy_test.go#L745-L757

Reply:
If you want to check whether the value is actually retrieved by GetClusterStatus,
write the test for GetClusterStatus in a separate file.  Tests in `operators.go`
are meant to test operations such as `SchedulerRestartOp`, and `SchedulerRestartOp`
was tested unnecessarily too many times.